http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -121,6 +121,8 @@ class LeoPubsubMessageSubscriber[F[_]](
               e.getMessage
             )
           }
+        case _: CreateAppV2Message => F.unit
```

Comment:
Great to see PubSub support started! (even if TODO :)).

Eventually the azure pubsub handing should invoke methods on https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSAlgebra.scala